clusterloader/glide.yaml

```diff
@@ -0,0 +1,23 @@
+package: github.com/sjug/perf-tests/clusterloader
```

Comment:
The package path isn't correct as it refers to my repo, how should I reference it?